velox/functions/sparksql/Arithmetic.h

```diff
@@ -152,6 +152,14 @@ struct FloorFunction {
   }
 };
 
+template <typename T>
+struct AcosFunction {
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE void call(TInput& result, TInput a) {
+    result = std::acos(a);
```

Comment:
Looks like this implementation is the same as Presto. Would it make sense to reuse?

Reply:
Hi @mbasmanova thank you for your reply, I have reused presto's acos function implementation, I recreated a pull request([5601](https://github.com/facebookincubator/velox/pull/5601)), please help me review again, thank you.